game_board.py

```diff
@@ -230,19 +346,28 @@ def main(screen,player1_name,player2_name):
     
     # plus_list, coordinates of the "+" chars in the map_coordinates list 
     plus_list = make_plus_list(map_coordinates)
-     
+    
+    
+    #player_turn will determine which player that will start ,True for player1
+    player_turn = random_player_start()
+    
+    remaining_stones_p1 = 9
+    remaining_stones_p2 = 9
+    #Prints player start text once
+    print_once = 0
     while 1:
         screen.clear()
-
+        if print_once == 0: print_player_start(screen,player_turn,player1_name,player2_name)
```

Comment:
You can let the print_once be in the if statement. Otherwise it will be incremented every tick
```suggestion
        if print_once == 0: 
             print_player_start(screen,player_turn,player1_name,player2_name)
             print_once += 1
```

Reply:
Fixed